Registry/registry.WRF-CMAQ-twoway

```diff
@@ -0,0 +1,51 @@
+#state    real  sw_ttauxar_01       ij             misc                     1    -      hr     "SW_TTAUXAR_01"   "Optical depth sum of SW_GTAUXAR_01" ""
+#state    real  sw_ttauxar_02       ij             misc                     1    -      hr     "SW_TTAUXAR_02"   "Optical depth sum of SW_GTAUXAR_02" ""
+#state    real  sw_ttauxar_03       ij             misc                     1    -      hr     "SW_TTAUXAR_03"   "Optical depth sum of SW_GTAUXAR_03" ""
+#state    real  sw_ttauxar_04       ij             misc                     1    -      hr     "SW_TTAUXAR_04"   "Optical depth sum of SW_GTAUXAR_04" ""
+#state    real  sw_ttauxar_05       ij             misc                     1    -      hr     "SW_TTAUXAR_05"   "Optical depth sum of SW_GTAUXAR_05" ""
+#state    real  sw_asy_fac_01       ikj            misc                     1    -      hr     "SW_ASY_FAC_01"   "Corresponding Asymmetry Factor of SW_GTAUXAR_01" ""
+#state    real  sw_asy_fac_02       ikj            misc                     1    -      hr     "SW_ASY_FAC_02"   "Corresponding Asymmetry Factor of SW_GTAUXAR_02" ""
+#state    real  sw_asy_fac_03       ikj            misc                     1    -      hr     "SW_ASY_FAC_03"   "Corresponding Asymmetry Factor of SW_GTAUXAR_03" ""
+#state    real  sw_asy_fac_04       ikj            misc                     1    -      hr     "SW_ASY_FAC_04"   "Corresponding Asymmetry Factor of SW_GTAUXAR_04" ""
+#state    real  sw_asy_fac_05       ikj            misc                     1    -      hr     "SW_ASY_FAC_05"   "Corresponding Asymmetry Factor of SW_GTAUXAR_05" ""
+#state    real  sw_ssa_01           ikj            misc                     1    -      hr     "SW_SSA_01"       "Corresponding Single scattering Albedo of SW_GTAUXAR_01" ""
+#state    real  sw_ssa_02           ikj            misc                     1    -      hr     "SW_SSA_02"       "Corresponding Single scattering Albedo of SW_GTAUXAR_02" ""
+#state    real  sw_ssa_03           ikj            misc                     1    -      hr     "SW_SSA_03"       "Corresponding Single scattering Albedo of SW_GTAUXAR_03" ""
+#state    real  sw_ssa_04           ikj            misc                     1    -      hr     "SW_SSA_04"       "Corresponding Single scattering Albedo of SW_GTAUXAR_04" ""
+#state    real  sw_ssa_05           ikj            misc                     1    -      hr     "SW_SSA_05"       "Corresponding Single scattering Albedo of SW_GTAUXAR_05" ""
+
+#state    real  sw_zbbcddir         ij             misc                     1    -      hr     "SW_ZBBCDDIR"     "Clear sky downward direct shortwave flux" "W m-2"
+#state    real  sw_dirdflux         ij             misc                     1    -      hr     "SW_DIRDFLUX"     "Direct downward shortwave surface flux" "W m-2"
+#state    real  sw_difdflux         ij             misc                     1    -      hr     "SW_DIFDFLUX"     "Diffuse downward shortwave surface flux" "W m-2"
```

Comment:
@dwongepa 
Is there a reason why these are all commented out?

We probably need to have a CMAQ package variable for when they are uncommented.